packages/ui/src/components/va-data-table/hooks/useRows.ts

```diff
@@ -18,17 +18,17 @@ export interface TableRow {
   cells: TableCell[];
 }
 
-const buildTableCell = (rowIndex: number, column: TableColumn, source: any): TableCell => ({
+const buildTableCell = (rowIndex: number, column: TableColumn, source: ITableItem): TableCell => ({
```

Comment:
`source` is definitely no the best name for this thing. It's a `rowSource` (or something similar). But `source` is a confusing name. It seems as though a table row is a source data for a row cell.

Reply:
Fixed.